cluster/src/main/scala/org/apache/spark/executor/SnappyExecutor.scala

```diff
@@ -108,6 +108,24 @@ class SnappyExecutor(
       }
     }
   }
+
+  override def isStoreCloseException(t : Throwable) : Boolean = {
+    try {
+      Misc.checkIfCacheClosing(t)
+    } catch {
+      case ex: CacheClosedException => true
+      case _ => false
+    }
+    return false;
+  }
+
+  override def isStoreException(t: Throwable): Boolean = {
+    if (GemFireXDUtils.retryToBeDone(t)) {
+      return true;
```

Comment:
just return "GemFireXDUtils.retryToBeDone(t)" and doesn't need another if-then check